src/riak_kv_ts_svc.erl

```diff
@@ -490,11 +490,11 @@ sub_tsgetreq(Mod, DDL, #tsgetreq{table = Table,
             %% the columns stored in riak_object are just
             %% names; we need names with types, so:
             ColumnTypes = get_column_types(ColumnNames, Mod),
-            {reply, {tsgetresp, {ColumnNames, ColumnTypes, [Row]}}, State};
+            {reply, #tsgetresp{columns={ColumnNames, ColumnTypes}, rows=[Row]}, State};
         {error, {bad_key_length, Got, Need}} ->
             {reply, key_element_count_mismatch(Got, Need), State};
         {error, notfound} ->
-            {reply, make_rpberrresp(?E_NOTFOUND, "notfound"), State};
+            {reply, #tsgetresp{}, State};
```

Comment:
Why is this being changed back? I changed the Python and Node.js test suites to take the error into account.